bench/Benchmark.hs

```diff
@@ -0,0 +1,119 @@
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Benchmark (exbudget and script size) for Plutus scripts
+module Benchmark (
+  -- | * Types
+  Benchmark (..),
+  NamedBenchmark (..),
+  ScriptSizeBytes (..),
+  -- | * Benchmark an arbitraty Plutus script
+  benchmarkScript,
+  -- | * Benchmark entrypoints
+  bench,
+  benchGroup,
+  benchMain,
+) where
+
+import Codec.Serialise qualified as Codec
+import Control.Arrow ((&&&))
+import Data.ByteString.Lazy qualified as BSL
+import Text.PrettyPrint.Boxes qualified as B
+
+import Data.ByteString.Lazy qualified as LB
+import Data.ByteString.Short qualified as SBS
+import Data.Csv (
+  DefaultOrdered (..),
+  ToField,
+  ToNamedRecord (..),
```

Comment:
We generally don't want to do these kinds of imports. It makes it unclear where things are coming from.

Reply:
Can you add these kinds of expectations to https://github.com/Plutonomicon/plutarch/issues/107 ?